src/Frontend/Modules/MailMotor/Actions/Unsubscribe.php

```diff
@@ -56,7 +56,7 @@ public function execute()
         if (!$form->isValid()) {
             $this->tpl->assign('form', $form->createView());
 
-            $this->tpl->assign('mailMotorUnsubscriptionHasFormError', true);
+            $this->tpl->assign('mailMotorUnsubscribeHasFormError', true);
```

Comment:
isn't this wrong? because if the form hasn't been submitted it will also come in here